src/helpers/AllureHelper.ts

```diff
@@ -0,0 +1,26 @@
+import type { Page } from '@playwright/test';
+import { allure } from 'allure-playwright';
+
+export class AllureHelper {
+  
+  convertCamelCaseToSpaceSeparatedWords(inputString: string) {
+    return inputString.replace(/([a-z])([A-Z])/g, '$1 $2');
+  };
+
+  async parseComponentAndFeatureFromFilePath(specFilePath: string): Promise<Record<string, string>> {
+    const normalizedPath = specFilePath.replace(/\\/g, '/');
+    const parts = normalizedPath.split('/');
+    const componentName = parts[parts.length - 2];
+    const featureNameWithExt = parts[parts.length - 1];
+    const component: string = this.convertCamelCaseToSpaceSeparatedWords(componentName);
```

Comment:
When you assign the value right away after defining the const, you can omit the type definition. 

  const component = this.convertCamelCaseToSpaceSeparatedWords(componentName);

Reply:
fixed